src/applications/search/containers/SearchApp.jsx

```diff
@@ -179,6 +179,9 @@ class SearchApp extends React.Component {
     return (
       <p>
         Showing {totalEntries === 0 ? '0' : `${resultRangeStart}-${resultRangeEnd}`} of {totalEntries} results
+        <span className="usa-sr-only">
+          {' '}for "{this.props.router.location.query.query}"
```

Comment:
Can use ES6 string interpolation if you don’t want to add the empty space that prettier complains about